test/Fields/Lens.hsc

```diff
@@ -0,0 +1,52 @@
+module Fields.Lens where
```

Comment:
Is this file still needed?

Reply:
It's a test for whether all fields are prefixed correctly, not a library bit. Unless that somehow adds `template-haskell` to the build process for users, but I don't think it does.

Reply:
I think depending on `template-haskell` in the test-suite is OK. I'm on the fence about whether or not it's needed though, since there shouldn't really be a connection between this library and `freetype-lens`, but given that it's not in the main library I'm inclined to keep it.